Gem/Code/Source/Components/Multiplayer/EnergyBallComponent.cpp

```diff
@@ -282,7 +274,21 @@ namespace MultiplayerSample
 
         RPC_BallExplosion(m_hitEvent);
 
-        Multiplayer::GetNetworkEntityManager()->MarkForRemoval(GetNetBindComponent()->GetEntityHandle());
+        // Wait 5 seconds before cleaning up the entity so that the explosion effect has a chance to play out
+        // Capture just the netEntityId in case we have a level change or some other operation that clears out entities before our lamda triggers
```

Comment:
```suggestion
        // Capture just the netEntityId in case we have a level change or some other operation that clears out entities before our lambda triggers
```